public/index.html

```diff
@@ -59,15 +59,19 @@
           style="position: absolute; left: 0; top: 0; display: flex; flex-direction: column; height: 100vh; width: 100vw; background-color: #fafbfc; font-family: 'Montserrat', sans-serif; font-variant-numeric: tabular-nums; -webkit-font-smoothing: antialiased; -moz-osx-font-smoothing: grayscale;">
         <div
             style="flex-grow: 1; color: rgb(51, 63, 82); font-size: 54px; display: flex; align-items: center; justify-content: center;">
-          <img role="img" src="%PUBLIC_URL%/loading-spinner.svg"
+          <img role="img" src="%PUBLIC_URL%/loading-spinner.svg" alt=""
                style="height: 54px; margin-right: 2rem; opacity: 0.5;">
           Loading Terra...
         </div>
         <div
             style="flex: 0 0 auto; display: flex; align-items: center; justify-content: space-between; height: 66px; padding: 0 1rem; background-color: rgb(109, 110, 112);">
-          <img role="img" src="%PUBLIC_URL%/logo-grey.svg"
+          <img role="img" src="%PUBLIC_URL%/logo-grey.svg" alt="Terra logo"
                style="height: 40px; margin-bottom: 4px;">
-          <div style="font-weight: 600; font-size: 10px; color: white;">Copyright ©2019</div>
+          <div style="font-weight: 600; font-size: 10px; color: white;" id="copyright"></div>
+          <script>
+            const buildTimestamp = new Date(parseInt('%REACT_APP_BUILD_TIMESTAMP%', 10))
+            window.document.getElementById('copyright').innerHTML = `Copyright ©${buildTimestamp.getFullYear()}`
+          </script>
```

Comment:
TOL: Having a special script for this seems like a lot of effort for something that's only on the screen briefly. A few thoughts:
* Is there a way to include logic in the HTML template, such that this is computed at compile time?
* Could we just remove the copyright here entirely?
* Should we just leave the number static and remember to bump it every year?

Reply:
- For rendering it in the template, we'd have to add a new `REACT_APP_` variable, but we could do it.
- It had been static, and we hadn't updated for 5 months, so that feels like a non-starter.
- We could just remove it, totally.

Reply:
I'm just gonna remove it, I don't think it's necessary.